.github/workflows/pull_request.yml

```diff
@@ -1,18 +1,39 @@
+name: Build Chain
+
 on: [ workflow_dispatch ]
 
 jobs:
-  build-chain:
+  build-chain-openjdk8:
+    runs-on: ubuntu-latest
+    name: Pull Request openjdk8
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build Chain
+      id: build-chain
+      uses: kiegroup/github-action-build-chain@openjdk8
+      with:
+        parent-dependencies: 'drools@7.x:master'
+        child-dependencies: 'jbpm@7.x:master'
+        build-command: 'mvn -e clean install -nsu -Prun-code-coverage,wildfly -Dfull  -Dintegration-tests=true -Dmaven.test.failure.ignore=true -DjvmArgs="-Xms1g -Xmx4g" | mvn -nsu generate-resources -Psonarcloud-analysis -DjvmArgs="-Xms1g -Xmx4g"'
```

Comment:
The profiles `run-code-coverage` and `sonarcloud-analysis` should be never in the same maven execution:
`mvn -e clean install -nsu -Prun-code-coverage` runs tests with measuring the coverage
`mvn validate -Psonarcloud-analysis` merges the coverage information into a single XML coverage file and triggers the sonarcloud

Reply:
Also, as I see the same command in the `build-chain-openjdk11`, SonarCloud should be triggered only once per any change on the PR. There is no point in running it inside multiple checks.

Reply:
Question: do the GitHub Actions inject SonarCloud auth token into the build? Last time I experimented with GA and SonarCloud, it didn't work because GA deliberately avoid injecting secrets into builds of PRs opened from a forked repo. The motivation was to protect the secret from being read by a malicious PR.

Reply:
thanks @rsynek `build-command` specifies multiple commands (maybe `build-commands` is a better name :thinking: ) in this case there are two commands split by `|` so two different maven executions. I already removed sonar execution from the rest of the projects but due to sonar execution is in `build-command` (only executed when the PR is for the current project) and the optaplanner event is `workflow_dispatch` this will never be triggered by a PR.
About sonarcloud credentials I already open a JIRA for it https://issues.redhat.com/browse/BXMSPROD-895
I'm going to remove sonarcloud execution anyway until BXMSPROD-895 is fixed.

Reply:
just one thing `with` variables don't allow arrays, that's why it's defined as a string split by `|`

Reply:
Thanks, I missed the `|` in the command string.

Reply:
sonar analysis command removed